site/_docs/reference.md

```diff
@@ -2654,6 +2654,7 @@ BigQuery's type system uses confusingly different names for types and functions:
 | p | expr :: type                                   | Casts *expr* to *type*
 | m | expr1 <=> expr2                                | Whether two values are equal, treating null values as the same, and it's similar to `IS NOT DISTINCT FROM`
 | * | ACOSH(numeric)                                 | Returns the inverse hyperbolic cosine of *numeric*
+| s | ARRAY()                                        | Construct an empty array in Apache Spark
```

Comment:
Couldn't this and the next line be unified?

Reply:
thanks for reviewing. IMHO, The empty array is special, and it will be highlighted here to make it clearer. Of course, we can merge it in the next line, but I may not want to modify the parameter form of `ARRAY(expr [, expr ]*)` and only add a description of an empty array. what do you think?

Reply:
I don't see any other case where the same function has two rows.

Reply:
> I don't see any other case where the same function has two rows.

make sense, just a small improvement. merged.